.circleci/test.sh

```diff
@@ -8,10 +8,6 @@ source .circleci/common.sh
 PYTORCH_DIR=/tmp/pytorch
 XLA_DIR=$PYTORCH_DIR/xla
 
-sudo apt-get -qq update
```

Comment:
Better to leave these to indicate the clang-9 dependencies. These commends would be no-ops anyway?

Reply:
Good point! Again, just part of the `revert` commit 😄 I'll be adding these back when I upgrade back to cuda11.